pkgs/top-level/python-packages.nix

```diff
@@ -145,6 +145,39 @@ in {
 
   automat = callPackage ../development/python-modules/automat { };
 
+  asana = buildPythonPackage rec {
```

Comment:
we do not longer add new expressions here, but in a separate file in python-modules, due the size of this file.

```
asana = callPackage ../development/python-modules/asana { };
```

see also: https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.md#contributing-guidelines